packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.js

```diff
@@ -60,17 +61,20 @@ const PaymentInformation = props => {
     const paymentInformation = doneEditing ? (
         <Summary onEdit={showEditModal} />
     ) : (
-        <PaymentMethods
-            onPaymentError={handlePaymentError}
-            onPaymentSuccess={handlePaymentSuccess}
-            resetShouldSubmit={resetShouldSubmit}
-            shouldSubmit={shouldSubmit}
-        />
+        <Form>
```

Comment:
Following up to @sirugh's question; I think this form was shared out of convenience, and I don't see that any controls or state were shared between the two. I would move this into `PaymentMethods` and it can probably just replace the root div.

Reply:
There are effects somewhere in the PaymentMethods tree that expect the Form context, so this is appropriate for now.